Sources/Nats/NatsConnection.swift

```diff
@@ -21,6 +21,10 @@ import NIOSSL
 import NIOWebSocket
 import NKeys
 
+#if canImport(FoundationNetworking)
```

Comment:
I think this can be quite obscure for users.
How about making it obvious clear condition for linux builds, maybe even specyfing what is needed on linux to make it work?